root/components/EntityTabs.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * @flow
+ * Copyright (C) 2017 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const Tabs = require('./Tabs.js');
+const EntityTabLink = require('./EntityTabLink');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const {ENTITIES} = require('../static/scripts/common/constants');
+
+const tabLinkNames = {
+  artists: l('Artists'),
+  cover_art: l('Cover Art'),
+  discids: l('Disc IDs'),
+  events: l('Events'),
+  fingerprints: l('Fingerprints'),
+  labels: l('Labels'),
+  map: l('Map'),
+  performances: l('Performances'),
+  places: l('Places'),
+  recordings: l('Recordings'),
+  releases: l('Releases'),
+  users: l('Users'),
+  works: l('Works'),
+};
+
+const buildLinkObject = (content, entity, subPath, page, pageName) => {
```

Comment:
s/buildLinkObject/buildLink/ maybe? It's shorter and I'm not sure `Object` adds any information. :) (You don't use `buildLinkObjects` below.)